cli/src/subcmd/me.rs

```diff
@@ -1,57 +1,168 @@
+
+    println!("Name:       {}", user_info.name);
+    println!("Status:     {}", user_info.status.unwrap_or_else(|| "<no status is set>".to_string()));
+    println!("Public key: {}", encode(user_info.keypair.public.as_bytes()));
+
+    println!("\nIdentities:");
+    for (name, id) in user_info.identities.iter() {
+        println!("  - {}: {}", name, id);
+    }
+
+    Ok(())
+}
+
+fn view(cmd: &ViewCmd) -> Result<()> {
+    let user_info = load_user_info()?;
+
+    match cmd {
+        ViewCmd::Name => {
+            println!("{}", user_info.name);
+        }
+        ViewCmd::Status => match user_info.status {
+            Some(status) => println!("{}", status),
+            None => println!("You haven't set a status!"),
+        },
+        ViewCmd::Pubkey => {
+            println!("{}", encode(user_info.keypair.public.as_bytes()));
+        }
+        ViewCmd::Identities => {
+            for (name, id) in user_info.identities.iter() {
+                println!("{}: {}", name, id);
+            }
+        }
+    }
+    Ok(())
+}
+
+fn edit(opts: &EditOpts) -> Result<()> {
     let mut user_info = load_user_info()?;
 
     match opts {
-        StatusOpts::Get => {
-            match user_info.status {
-                Some(status) => println!("{}", status),
-                None => println!("You haven't set a status!"),
+        EditOpts::Name { name } => {
+            user_info.name = name.clone();
+        }
+        EditOpts::Status(opts) => {
+            user_info.status = opts.status.clone();
+        }
+        EditOpts::Identities(opts) => match opts {
+            IdentitiesOpts::Add { id, name } => {
+                if user_info.identities.contains_key(name) {
+                    return Err(Error::from(format!("identity '{}' already exists, use modify instead", name)));
+                }
+                user_info.identities.insert(name.clone(), id.clone());
+            }
+            IdentitiesOpts::Modify { id, name } => {
+                if !user_info.identities.contains_key(name) {
+                    return Err(Error::from(format!("identity '{}' does not exist, use add instead", name)));
+                }
+                user_info.identities.insert(name.clone(), id.clone());
+            }
+            IdentitiesOpts::Remove { name } => {
+                match user_info.identities.remove(name) {
+                    Some(_) => {},
+                    None => return Err(Error::from(format!("can't remove identity '{}', identity does not exist!", name))),
+                };
             }
         },
-        StatusOpts::Set { status } => {
-            user_info.status = Some(status.clone());
-            write_user_info(&user_info)?;
-        },
-        StatusOpts::Clear => {
-            user_info.status = None;
-            write_user_info(&user_info)?;
-        }
     }
+    write_user_info(&user_info)?;
+    println!("Updated configuration, don't forget to (re)publish me.frauth!");
```

Comment:
```suggestion
    println!("Updated configuration, don't forget to (re)publish with `frauth publish`!");
```

It might be better to mention the command, as not all users will use `me.frauth` (I'm more worried about them going and finding the User Info file, which is also called `me.frauth`).